src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -38,65 +39,30 @@ const generateNGrams = () => {
 </script>
 
 <template>
-    <div class="ma-keywords-generator">
-        <div class="ma-header">
-            <label for="allwords">Keyword Generator</label>
-        </div>
-        <MaInput id="allwords" v-model:value = "inputText" type="textarea" hint-text="Please select how you want to see n-grams and dont forget to use ',' for different exluded words." style="height: 100px; resize: none; border-radius: 10px; margin: 5px; width: 100%;"> </MaInput>
-        <div style="margin: 10px">Checked N-Grams: {{ selectedNGrams }}</div>
-        <div style="display: flex; margin: 10px">
-            <div v-for="index in Math.min(filteredWords.length, 10)" :key="index" >
-                <input type="checkbox" :id="'n' + index" :value="index" v-model="selectedNGrams" />
-                <label :for="'n' + index">{{ index }}</label>
-            </div>
-        </div>
-        <label for="excludeWords">Exclude Words:</label>
-        <input type="text" id="excludeWords" v-model="excludeWords" placeholder="Comma separated">
-        <button class="submit-button" style="margin-top: 10px;" @click="generateNGrams">Submit</button>
-        <div class="preview" v-for ="(keyword, index) in nGrams" :key="index" >
-            <strong>{{ keyword.n + "-gram:" }}</strong>
-            <div v-for="tag in keyword.combinations" style="width: fit-content; margin: 2px;">
-                <ma-badge shape="square" variant="dark" type="secondary" size="medium"><ma-badge shape="dot" variant="dark"/> {{ tag }} </ma-badge>
-            </div>
-        </div>
-    </div>
+  <div class="ma-keywords-generator p-4">
+      <div class="ma-header mb-4">
+          <label for="allwords" class="text-xl font-bold">Keyword Generator</label>
+      </div>
+      <MaInput id="allwords" v-model:value="inputText" type="textarea" hint-text="Please select how you want to see n-grams and don't forget to use ',' for different excluded words." class="input-area resize-none w-full p-2 mb-1" style="resize: none;"></MaInput>
```

Comment:
Please don't have so many items on a single line. For readability, let's move to a new line after each prop.